test/unit/mock/mock_http_client.js

```diff
@@ -40,8 +40,17 @@ MockHttpClient.prototype.request = function (request)
       buildRequestToOutputMap(buildRequestOutputMappings(this._clientInfo));
   }
 
+  // Closing a session includes a unique uuid for requestID and requestGUID as query parameters in the url
```

Comment:
Where do we draw the conclusion of `requestGUID`?

Reply:
I'm not aware why .NET connector includes a requestGUID, but since JDBC connector doesn't use it I could remove the requestGUID parameter on node so it will have parity with JDBC